### DISCUSSION TITLE
What are the minimum requirements to self host supabase?

### DISCUSSION THREAD

**Discussion by LawMixer** [supabase/supabase#21132] [Questions]
As the title suggest, I'm looking for the minimum requirements to self host supabase. How much resources should I allocate to it? Yes, it's a database, so it'll depend on how much information I got, though what is the minimum to get the application to startup and be smooth?

Thanks!

**Comment by Sukikiroi** [comment]
Hello sir , this might Hellp You


refs : =>https://www.reddit.com/r/Supabase/comments/1aydpyg/minimum_specs_for_supabase_self_hosted/

**Reply by zotodev** [reply]
The link you shared, just has the question and no response.

**Reply by quentin-aslan** [reply]
Yes there are no response
